src/wordle/controller.py

```diff
@@ -55,4 +55,6 @@ def check_wordle_state(self) -> bool:
         """
         check if wordle is over
         """
-        return self.driver.find_elements(By.CLASS_NAME, "Stats-module_statsContainer__g23s0")
+        return self.driver.find_elements(
+            By.CLASS_NAME, "Stats-module_statsContainer__g23s0"
```

Comment:
Maybe for all these class names we can add an enum that is like CONNECTIONS.CORRECT_TILE etc.

Reply:
sounds good I can do that right now